spec/controllers/api/users_controller_spec.rb

```diff
@@ -23,9 +24,32 @@
     end
 
     it 'should have properties for user 1' do
-      user = result.detect { |u| u['id'] == user_1.id }
+      user = result.detect { |u| u['id'].to_i == user_1.id }
       expect(user['name']).to eq(user_1.name)
       expect(user['email']).to eq(user_1.email)
     end
   end
+
+  describe "POST create" do
+    let!(:user_1) { FactoryGirl.create(:user) }
+    before do
+      @request.env['HTTP_AUTHORIZATION'] = ActionController::HttpAuthentication::Basic.encode_credentials(user_1.username, user_1.password)
```

Comment:
Line is too long. [139/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.